onnxruntime/core/optimizer/embed_layer_norm_fusion.cc

```diff
@@ -77,6 +77,19 @@ static void AddNodes(std::vector<NodeIndex>& node_indices,
   }
 }
 
+static bool IsNeighborNodeExpectedTypes(Node::NodeConstIterator start, const Node::NodeConstIterator end, const std::vector<std::string>& expected_types) {
+  if (expected_types.size() == 0) {
```

Comment:
nit: this check of expected_types can be removed at least for current usage (since the only inference has size of 2).